lib/schema/document_types.rb

```diff
@@ -37,7 +37,11 @@ def parse
     end
 
     fields = Hash[field_names.map { |field_name|
-      [field_name, field_definitions.get(field_name)]
+      field_definition = field_definitions[field_name]
```

Comment:
I think it would be better to use `fetch` for this:

```ruby
field_definition = field_definitions.fetch(field_name) {
  raise_error(%{Undefined field "#{field_name}"})
}
```